src/data/resolvers/user.ts

```diff
@@ -50,6 +51,78 @@ const getUserReputation = async (
   return reputationAmount;
 };
 
+const getUserStakedBalance = async (
+  colonyManager: ColonyManager,
+  colonyAddress: Address,
+  walletAddress: Address,
+): Promise<BigNumber> => {
+  const votingReputationClient = await colonyManager.getClient(
+    ClientType.VotingReputationClient,
+    colonyAddress,
+  );
+  /**
+   * @NOTE If there will be more staking events
```

Comment:
You mean different event than `MotionStaked`, right ?

Reply:
yes, more ways of token staking (new events)